src/encoder.rs

```diff
@@ -3449,9 +3419,37 @@ pub fn encode_show_existing_frame<T: Pixel>(
   fi: &FrameInvariants<T>, fs: &mut FrameState<T>, inter_cfg: &InterConfig,
 ) -> Vec<u8> {
   debug_assert!(fi.show_existing_frame);
+  let obu_extension = 0 as u32;
```

Comment:
Any reason for not using `0u32` ?

Reply:
Not familiar enough with Rust, so i wasn't aware that's an option

Reply:
It can be `0`, the type should inferred w/out external help.